### PR TITLE
UX color changes for graph

### PR DESCRIPTION
**Pull request by jshaughn**
- gray background
- white cy toolbar buttons
- consistent use of successGreen/warnOrange/dangerRed color and terminology
  - nodes
  - edges
  - side panel charts
    - change http 4xx orange to dark red to limit confusion with warning (they are errors)
  - legend svgs and text
    - rename some svgs for consistency

Fixes https://github.com/kiali/kiali/issues/1760

These changes are based on this UX rendering: https://marvelapp.com/41jgfd4/screen/59366802, and also the updated PF4 color schemes.

@serenamarie125 , @josejulio , please look at the images below, any feedback?

Before:
image

After:
image

One thing I see in the "After" image is that with the lighter colored edges, and the new background, the edges, especially green, are a little hard to see.  Here is an after shot with edge widths increased from 1 to 2 pixels:
image

To be honest, even without the color changes, it's more in proportion with the larger nodes.  The selected edge would also add a pixel:
image

### CODE REVIEW

File: src/components/Pf/PfColors.tsx [hunk 1]
```diff
@@ -86,13 +86,25 @@ export enum PfColors {
...
 
   // Kiali colors that use PF colors
-  Gray = '#72767b' // Black600
+
+  Gray = Black600,
+  GrayBackground = Black150,
+
+  // Health/Alert colors https://www.patternfly.org/v4/design-guidelines/styles/colors
+  Danger = '#C9190B',
+  DangerBackground = '#470000',
```
Comment (josejulio) [inline src/components/Pf/PfColors.tsx]: `Red400`

File: src/components/Pf/PfColors.tsx [hunk 2]
```diff
@@ -86,13 +86,25 @@ export enum PfColors {
...
+  Info = '#73BCF7',
+  InfoBackground = '#004368',
```
Comment (josejulio) [inline src/components/Pf/PfColors.tsx]: `Blue600`

File: src/components/Pf/PfColors.tsx [hunk 3]
```diff
@@ -86,13 +86,25 @@ export enum PfColors {
...
+  Success = '#92D400',
```
Comment (josejulio) [inline src/components/Pf/PfColors.tsx]: `LightGreen400`

File: src/components/Pf/PfColors.tsx [hunk 4]
```diff
@@ -86,13 +86,25 @@ export enum PfColors {
...
+  SuccessBackground = '#486B00',
```
Comment (josejulio) [inline src/components/Pf/PfColors.tsx]: `LightGreen600`

File: src/components/Pf/PfColors.tsx [hunk 5]
```diff
@@ -86,13 +86,25 @@ export enum PfColors {
...
+  Warning = '#f0AB00',
```
Comment (josejulio) [inline src/components/Pf/PfColors.tsx]: `Gold400`

File: src/components/Pf/PfColors.tsx [hunk 6]
```diff
@@ -86,13 +86,25 @@ export enum PfColors {
...
+  WarningBackground = '#795600'
```
Comment (josejulio) [inline src/components/Pf/PfColors.tsx]: `Gold600`

**Comment by josejulio** [comment]
I'm not sure if it makes sense to use the other enums instead of writing the color string again, but left a comment with the ones i found in your changes. 
I see that you updated 
`Gray = '#72767b' // Black600` to `Gray = Black600` so maybe yes

File: src/components/Pf/PfColors.tsx (hunk 1 above)
Comment (jshaughn) [reply src/components/Pf/PfColors.tsx]: Ah, yeah, I copied the hex from the UX page and forgot to match them with he constants they represent.  I'll update.

**Comment by abonas** [comment]
How the node colors red/grey/orange work with color blindness? If the color is the only indication then it is not a good practice. Cc @serenamarie125

**Comment by jshaughn** [comment]
> How the node colors red/grey/orange work with color blindness?

@abonas @serenamarie125 , This PR is only changing colors to be in line with both PF4 and also the UX topology design for the graph.  Color blindness concerns are taken into account by the choice of foreground and background choices, but also by the changes in edge styling introduce in the latest design.  Edge labeling and the side panel show textual info as well.  Again, color blindness is orthogonal to this PR, which only applies the designated coloring.

**Comment by josejulio** [comment]
> One thing I see in the "After" image is that with the lighter colored edges, and the new background, the edges, especially green, are a little hard to see. Here is an after shot with edge widths increased from 1 to 2 pixels:

+1

IMO we should increase the size, now that we have bigger nodes the edges look really thin.

**Comment by jshaughn** [comment]
>  i think the background of the sidebar should also be gray

+1, that gap doesn't look good

**Comment by jshaughn** [comment]
@josejulio Latest commit fixes you issue and does a few other things, here is the commit message:

PR Feedback, Josejulio: Fix issue with white background behind side panel
- Apply Gray background to entire GraphPage
- Per UX design:
  - Apply white background to legend
  - Apply White background to all side panel header areas
  - Bring legend and side panel to top of page
- apply consistent border to legend and side panel

Here is the "after"shot:
image

File: src/pages/Graph/SummaryPanelCommon.tsx [hunk 7]
```diff
@@ -11,13 +11,18 @@ import { Response } from '../../services/Api';
...
 
 export enum NodeMetricType {
   APP = 1,
   WORKLOAD = 2,
   SERVICE = 3
 }
 
+export const summaryHeader: React.CSSProperties = {
+  backgroundColor: PfColors.White
+};
```
Comment (josejulio) [inline src/pages/Graph/SummaryPanelCommon.tsx]: We generally prefer [typestyle](https://github.com/typestyle/typestyle) over the inline css that this generates. see https://github.com/kiali/kiali-ui/blob/master/src/components/CytoscapeGraph/graphs/GraphStyles.ts#L65

**Comment by josejulio** [comment]
> @josejulio Latest commit fixes you issue and does a few other things, here is the commit message:
> 
> PR Feedback, Josejulio: Fix issue with white background behind side panel
> 
> * Apply Gray background to entire GraphPage
> * Per UX design:
>   
>   * Apply white background to legend
>   * Apply White background to all side panel header areas
>   * Bring legend and side panel to top of page
> * apply consistent border to legend and side panel

Nice job!

File: src/pages/Graph/SummaryPanelCommon.tsx (hunk 7 above)
Comment (jshaughn) [reply src/pages/Graph/SummaryPanelCommon.tsx]: +1 and I tried that but could not figure out how to combine a static className with a typestyle definition.  Feel free to make that change if you know how to do it, but I'm going to merge now that we have QE approval.

**Pull request merged**